mysql-test/suite/galera/r/galera_sync_wait_upto.result

```diff
@@ -5,39 +5,22 @@ INSERT INTO t1 VALUES (1);
 SELECT WSREP_SYNC_WAIT_UPTO_GTID(NULL);
 ERROR HY000: Incorrect arguments to wsrep_sync_wait_upto_gtid
 SELECT WSREP_SYNC_WAIT_UPTO_GTID('a');
-ERROR HY000: Incorrect arguments to wsrep_sync_wait_upto_gtid
+ERROR HY000: Could not parse GTID list
 SELECT WSREP_SYNC_WAIT_UPTO_GTID(2);
+ERROR HY000: Could not parse GTID list
+SELECT WSREP_SYNC_WAIT_UPTO_GTID('1-1-1,1-1-2');
```

Comment:
Having `WSREP_SYNC_WAIT_UPTO_GTID` as well as the existing `MASTER_GTID_WAIT` is going to be a annoyance for developers. They will need to be aware of the replication topologies when writing their application. If `WSREP_SYNC_WAIT_UPTO_GTID` can be removed and `MASTER_GTID_WAIT` function identically in both async and galera replication that would be a large improvement.

Reply:
MASTER_GTID_WAIT works in only master - slave architecture. WSREP_SYNC_WAIT_UPTO_GTID could be used in cluster on all nodes even when this master - slave connection is not defined. Constraint is that WSREP_SYNC_WAIT_UPTO_GTID only works for events originating from cluster itself, domain-server combination must match.

Reply:
`MASTER_GTID_WAIT`, while a little misnamed in a galera context, provides a good common interface.

With `wsrep_gtid_domain_id` being different it seems there's already a cluster GTID differentiation.